encord/user_client.py

```diff
@@ -418,6 +435,64 @@ def __upload_cvat_images(
     def get_cloud_integrations(self) -> List[CloudIntegration]:
         return self.querier.get_multiple(CloudIntegration)
 
+    def get_ontologies(
+        self,
+        title_eq: Optional[str] = None,
+        title_like: Optional[str] = None,
+        desc_eq: Optional[str] = None,
+        desc_like: Optional[str] = None,
+        created_before: Optional[Union[str, datetime]] = None,
+        created_after: Optional[Union[str, datetime]] = None,
+        edited_before: Optional[Union[str, datetime]] = None,
+        edited_after: Optional[Union[str, datetime]] = None,
+    ) -> List[Dict]:
+        """
+        List either all (if called with no arguments) or matching ontologies the user has access to.
+
+        Args:
+            title_eq: optional exact title filter
+            title_like: optional fuzzy title filter; SQL syntax
+            desc_eq: optional exact description filter
+            desc_like: optional fuzzy description filter; SQL syntax
+            created_before: optional creation date filter, 'less'
+            created_after: optional creation date filter, 'greater'
+            edited_before: optional last modification date filter, 'less'
+            edited_after: optional last modification date filter, 'greater'
+
+        Returns:
+            list of (role, projects) pairs for ontologies matching filter conditions.
```

Comment:
It seems to be like the return is a list of dict and I think we want to move away from using dicts in this way so can you please define a dataclass with the `ontology` and `user_role` members so this is typed?